__tests__/app.test.js

```diff
@@ -115,8 +121,55 @@ describe('/api/articles', () => {
             expect(body.msg).toBe('Bad Request')
         })
     });
-})
+});
 
 
 
+describe('/api/articles/:article_id/comments', () => {
+    test('status:400 returns all comments for article' , () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments.length).toBe(11)
+            body.comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number');
+                expect(typeof comment.votes).toBe('number');
+                expect(typeof comment.created_at).toBe('string');
+                expect(typeof comment.author).toBe('string');
+                expect(typeof comment.body).toBe('string');
+                expect(typeof comment.article_id).toBe('number');
```

Comment:
You know that all these comments will have article id 1 so I would specify that here